async_fuse/src/test/test_util.rs

```diff
@@ -51,17 +49,22 @@ pub fn setup(mount_path: impl AsRef<Path>) -> anyhow::Result<JoinHandle<()>> {
     Ok(th)
 }
 
-pub fn teardown(mount_dir: impl AsRef<Path>, th: JoinHandle<()>) -> anyhow::Result<()> {
+pub fn teardown(mount_dir: &Path, th: JoinHandle<()>) -> anyhow::Result<()> {
     info!("begin teardown");
     let seconds = 1;
     debug!("sleep {} seconds for teardown", seconds);
     thread::sleep(Duration::new(seconds, 0));
 
     smol::block_on(async {
-        mount::umount(&mount_dir).await.unwrap(); // TODO: remove unwrap()
+        mount::umount(mount_dir).await.unwrap_or_else(|err| {
```

Comment:
As we don't needthe unmount return value, is "expect" better than "unwrap_or_else"?

Reply:
Clippy has a lint error on `expect()`